environmentConstants.js

```diff
@@ -0,0 +1,6 @@
+
+const staticDataEnvironment = "staticData";
```

Comment:
mock environment could be the best naming for this purpose I guess, and we can also update the command as " npm run build-mock". What do you think?

Reply:
I'm not sure: "Mock. Mocks are the objects that store method calls. It referred to as the dynamic wrappers for dependencies used in the tests."
I think a better idea would be to change the defaultMockData.json file to defaultStaticData.json, so we keep the same naming.

Reply:
hmm, I am fine with that as well, it is your decision then :)